Python/pystate.c

```diff
@@ -78,6 +78,9 @@ _Py_thread_local PyThreadState *_Py_tss_tstate = NULL;
    also known as a "gilstate." */
 _Py_thread_local PyThreadState *_Py_tss_gilstate = NULL;
 
+/* The interpreter of the attached thread state */
```

Comment:
```suggestion
/* The interpreter of the attached thread state. */
```
It would also be helpful to note that this is an optimization and should be identical to `_Py_tss_tstate->interp`.

Reply:
done